app/src/js/actions/index.js

```diff
@@ -781,8 +781,6 @@ export const deleteRule = (ruleName) => ({
 
 export const enableRule = (payload) => {
   const rule = clonedeep(payload);
-  delete rule['queriedAt'];
-  delete rule['timestamp'];
 
   if (!rule.rule.value) {
```

Comment:
This is still necessary I think because of a minor bug in our rules update endpoint. I can investigate that as part of this ticket